store/copr/batch_coprocessor.go

```diff
@@ -329,8 +342,9 @@ func balanceBatchCopTask(ctx context.Context, kvStore *kvStore, originalTasks []
 			storeTaskMap[taskStoreID] = batchTask
 		}
 	} else {
-		stores := cache.RegionCache.GetTiFlashStores(tikv.LabelFilterNoTiFlashWriteNode)
-		aliveStores := filterAliveStores(ctx, stores, ttl, kvStore)
+		usedStores := getUsedStores(cache, usedTiFlashStoresMap)
+		logutil.BgLogger().Info("detecting available mpp stores")
```

Comment:
Looks like this log is duplicated since `filterAliveStores` will also record this log